Realm.podspec

```diff
@@ -52,7 +52,8 @@ Pod::Spec.new do |s|
   s.osx.vendored_library    = 'core/librealm.a'
 
   s.subspec 'Headers' do |s|
-    s.source_files          = 'include/**/*.h'
+    s.source_files          = 'include/**/*.{h,hpp}'
```

Comment:
Why do the `*.hpp` files need to have `project` visibility?  They should be private.